src/options/Main/OptionsList/ThemeSelection.jsx

```diff
@@ -0,0 +1,39 @@
+import { h } from 'preact';
+
+// import { Component, h } from 'preact';
+import 'material-components-web/dist/material-components-web.css';
+
+const ThemeSelection = function ThemeSelection({
```

Comment:
Formatting here probably shouldn't be so indented 😉

Reply:
ok, I've re-formatted by hand, but this was just the default formatting by WebStorm. I'm not sure what setting to change to make this persist, so please bear with me if something like this comes back -- I'm a big fan of setting up auto-formatter rules and then just using them, instead of chasing formatting by hand (:

Reply:
We should be using Prettier in this project so you may want to look into setting that up for WebStorm to avoid manually fixing formatting.